regression/goto-instrument-unwind/unwind-chain.sh

```diff
@@ -6,22 +6,18 @@ goto_instrument=$src/goto-instrument/goto-instrument
 cbmc=$src/cbmc/cbmc
 
 function usage() {
-  echo "Usage: chain unwind-num test_file.c"
+  echo "Usage: chain <options> <testfile>"
```

Comment:
If the testfile were required to be the first (instead of the last) argument then the code could be simplified and all the quotes would become unnecessary. More below.